client/src/questions-n-answers/sub-components/mini-components/addAnswerTable.jsx

```diff
@@ -3,14 +3,15 @@ import propTypes from 'prop-types';
 
 
 const AddAnswer = (props) => {
+  console.log(props)
```

Comment:
Unless valuable to everyone, might be good for us to comment out our console logs before we make PR-- otherwise can slightly tiresome to track down where all the logs are coming from and comment them out on our locals